Update average run time (#430)
exercism/docs: building/configlet/lint.md

  - If the track is `d` or `plsql`, the `"files.solution"` and `"files.test"` files _can_ overlap
  - The `"files.example` and `"files.exemplar"` files _can_ overlap
- The `"test_runner.average_run_time"` key is required if `"status.test_runner"` is equal to `true`

> - The `"test_runner.average_run_time"` value must be positive integer > 0 or a floating-point number > 0.0 with one decimal point of precision

Reviewer: ```suggestion
- The `"test_runner.average_run_time"` value must be an integer > 0
```

I wrote:

> An alternative is to move to integer only

but I should've asked explicitly about that.

Asking explicitly: if we were planning an Exercism-wide PR to change every value to an int, and we don't think that an extra decimal place of precision is ever useful, should we just say integer? This will avoid some specific handling for this field, and some awkwardness in `configlet fmt`, as described in https://github.com/exercism/docs/pull/430#issuecomment-1431502339.

Author: > Asking explicitly: if we were planning an Exercism-wide PR to change every value to an int, and we don't think that an extra decimal place of precision is ever useful, should we just say integer? This will avoid some specific handling for this field, and some awkwardness in configlet fmt, as described in https://github.com/exercism/docs/pull/430#issuecomment-1431502339.

Yes, I would prefer that greatly!

Author: I'd be happy to create a cross track PR

Reviewer: > I'd be happy to create a cross track PR

Great. So can we apply the above suggested change and merge this PR?

Merged the corresponding change to configlet lint: https://github.com/exercism/configlet/commit/d2e04a5f7ed187f50374bd89c3cc16eef40c3d0d

Author: Yes. Are you around this afternoon for a release? I can do the PRs shortly after releasing.

Reviewer: Yes - sounds good.

It should be straightforward on the configlet side - the only other user-facing change in configlet since the last release is https://github.com/exercism/configlet/commit/1c020ea6693e4e6c67a86d5f1f5a65228a9be556.
